web/ui/react-app/src/pages/targets/JobPanel.module.css

```diff
@@ -0,0 +1,3 @@
+.container {
+  margin-top: -12px;
+}
```

Comment:
nit: Missing newline at end of file (I think our linter catches these already for TSX files, but maybe not for CSS)